src/formats/report.js

```diff
@@ -0,0 +1,46 @@
+const fs = require( 'fs-extra' );
```

Comment:
This file has a handful of linter issues, you can see them with `npm run lint:js` — `npm run format:js` should auto-fix most of them 🙂